### DISCUSSION TITLE
your project does not run on collab

### DISCUSSION THREAD

**Discussion by vladgohn** [collabora/WhisperSpeech#35] [General]
https://colab.research.google.com/github/collabora/WhisperSpeech/blob/5ab823856c5387c49b38dcb692e2cbbfaa5972f2/Inference%20example.ipynb#scrollTo=fdac8407

  Building wheel for openai-whisper (pyproject.toml) ... done
  Building wheel for encodec (setup.py) ... done
  Building wheel for lit (pyproject.toml) ... done
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
llmx 0.0.15a0 requires cohere, which is not installed.
llmx 0.0.15a0 requires openai, which is not installed.
torchdata 0.7.0 requires torch==2.1.0, but you have torch 2.0.1 which is incompatible.
torchtext 0.16.0 requires torch==2.1.0, but you have torch 2.0.1 which is incompatible.
torchvision 0.16.0+cu121 requires torch==2.1.0, but you have torch 2.0.1 which is incompatible.

**Comment by jpc** [comment]
Hey, I tested it just now and the generation works, despite the `pip` warnings. Could give it another try?
